intg-denonavr/simplecommand.py

```diff
@@ -356,21 +356,35 @@ async def _handle_core_command(self, cmd: str) -> ucapi.StatusCodes:
         # pylint: disable=R0915
         match cmd:
             case "OUTPUT_1":
-                await self._receiver.async_hdmi_output("HDMI1")
+                # TODO: Replace with commented code once https://github.com/ol-iver/denonavr/pull/337 is merged
+                await self.send_simple_command("VSMONI1")
+                # await self._receiver.async_hdmi_output("HDMI1")
             case "OUTPUT_2":
-                await self._receiver.async_hdmi_output("HDMI2")
+                # TODO: Replace with commented code once https://github.com/ol-iver/denonavr/pull/337 is merged
+                await self.send_simple_command("VSMONI2")
+                # await self._receiver.async_hdmi_output("HDMI2")
             case "OUTPUT_AUTO":
-                await self._receiver.async_hdmi_output("Auto")
+                # TODO: Replace with commented code once https://github.com/ol-iver/denonavr/pull/337 is merged
+                await self.send_simple_command("VSMONIAUTO")
+                # await self._receiver.async_hdmi_output("Auto")
```

Comment:
The hdmi output commands still don't work:
- `send_simple_commad` finds no match and returns `ucapi.StatusCodes.NOT_IMPLEMENTED` which is ignored here.
- this method is already called from `send_simple_commad`!
- replacing it with `self._send_command` makes it work